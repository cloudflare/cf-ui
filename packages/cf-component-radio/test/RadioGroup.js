const React = require('react');
const assertEqualJSX = require('assert-equal-jsx');
const RadioGroup = require('../src/RadioGroup');
const Radio = require('../src/Radio');

describe('RadioGroup', function() {
  it('should render', function() {
    assertEqualJSX(
      <RadioGroup
        value="option1"
        onChange={() => {}}
        options={[
          { label: 'Option 1', name: 'group-option-1', value: 'option1' },
          { label: 'Option 2', name: 'group-option-2', value: 'option2' }
        ]}
      />,
      // should equal
      <div className="cf-radio__group">
        <Radio
          label="Option 1"
          name="group-option-1"
          value="option1"
          checked={true}
          onChange={() => {}}
        />
        <Radio
          label="Option 2"
          name="group-option-2"
          value="option2"
          checked={false}
          onChange={() => {}}
        />
      </div>
    );
  });
});
