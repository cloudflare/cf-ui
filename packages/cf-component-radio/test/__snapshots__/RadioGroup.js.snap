// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="a b c"
/>
`;

exports[`should render 2`] = `
"
.a {
  text-align: left
}

.b {
  display: inline-block
}

.c {
  vertical-align: middle
}
"
`;
